repository_service_tuf/cli/admin/metadata.py

```diff
@@ -0,0 +1,520 @@
+# SPDX-FileCopyrightText: 2022-2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+import click
+from datetime import datetime, timedelta
+from rich import box, markdown, prompt, table
+from typing import Any, List
+from tuf.api.metadata import Metadata, Root
+from securesystemslib.exceptions import StorageError
+
+from repository_service_tuf.cli import console
+from repository_service_tuf.cli.admin import admin
+from repository_service_tuf.cli.admin.ceremony import _load_key, _save_payload
+
+from repository_service_tuf.constants import KeyType
+from repository_service_tuf.helpers.tuf import (
+    Roles,
+    RSTUFKey,
+    RootInfo
+)
+
+INTRODUCTION = """
+# Metadata Update
+
+The metadata update allows:
+- extending root expiration
+- modification of the keys used for signing (no matter if root keys or the
+online key),
+- changing of the root threshold value
```

Comment:
```suggestion
- changing Root threshold value
```
nit: Name of the metadata